### PR TITLE
feat: readme catalog and gallery

### PR DESCRIPTION
**Pull request by SolMendiola**
# :star: Feature

---

#### :pencil2: Description:

Added readme of catalog and gallery app.

---

#### iOS

image

image

### CODE REVIEW

File: catalog/README.md [hunk 1]
```diff
@@ -1,39 +1,26 @@
...
-and the Flutter guide for
-[developing packages and plugins](https://flutter.dev/developing-packages).
--->
-
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+Catalog is a Flutter package that provides a base implementation for UX/UI components based on the design template of Xmartlabs.
+It aims to simplify and expedite the development process for Flutter projects, ensuring a consistent and visually appealing user interface.
 
 ## Features
```
Comment (fedecor9) [inline catalog/README.md:6]: Maybe we can add something about the theme here

Comment (fedecor9) [reply catalog/README.md:6]: Also we can explain where each part is, kind of what we have in the template readme with the folder structure.
So it's easier to modify later on

File: catalog/README.md [hunk 2]
```diff
@@ -1,39 +1,26 @@
...
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+- Ready-to-use UI components inspired by Xmartlabs design principles.
```
Comment (mirland) [inline catalog/README.md:8]: main?
```suggestion
- Ready-to-use main UI components inspired by Xmartlabs design principles.
```

File: catalog/README.md [hunk 3]
```diff
@@ -1,39 +1,26 @@
...
 
-TODO: List prerequisites and provide or point to information on how to
-start using the package.
-
-## Usage
+### Installation
 
-TODO: Include short and useful examples for package users. Add longer examples
-to `/example` folder.
+Add the following line to your `pubspec.yaml` file:
```
Comment (mirland) [inline catalog/README.md:15]:
```suggestion
Integrate the package in your project, adding the following line to your `pubspec.yaml` file:
```

File: catalog/README.md [hunk 4]
```diff
@@ -1,39 +1,26 @@
...
 ```
 
-## Additional information
+### Contributing
+We welcome contributions! If you find a bug or have a feature request, please open an [issue].
 
-TODO: Tell users more about the package: where to find more information, how to
-contribute to the package, how to file issues, what response they can expect
-from the package authors, and more.
+[issue]: https://github.com/xmartlabs/flutter-template
```
Comment (mirland) [inline catalog/README.md:26]:
```suggestion
[issue]: https://github.com/xmartlabs/flutter-template/issues
```

File: catalog/gallery/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+- **Easy Sharing:** Share specific components or screens with the design team for review.
+
+
+## Design System Documentation
+For comprehensive documentation on our design system, including guidelines, principles, and usage instructions, please refer to [Design System].
+
+### Contributing
+We welcome contributions! If you find a bug or have a feature request, please open an [issue].
+
+[issue]: https://github.com/xmartlabs/flutter-template
```
Comment (mirland) [inline catalog/gallery/README.md:19]:
```suggestion
[issue]: https://github.com/xmartlabs/flutter-template/issues
```

**Review by mirland — APPROVED** [review]

**Pull request merged**